compiler/rustc_hir_typeck/src/coercion.rs

```diff
@@ -92,14 +92,16 @@ impl<'a, 'tcx> Deref for Coerce<'a, 'tcx> {
 
 type CoerceResult<'tcx> = InferResult<'tcx, (Vec<Adjustment<'tcx>>, Ty<'tcx>)>;
 
-struct CollectRetsVisitor<'tcx> {
-    ret_exprs: Vec<&'tcx hir::Expr<'tcx>>,
+pub struct CollectRetsVisitor<'tcx> {
```

Comment:
this isn't needed

Reply:
Done.

Reply:
Oh, shit, had in flight changes... let me clean up the mess first.